samples/bluetooth/iso_broadcast_benchmark/sample.yaml

```diff
@@ -4,5 +4,5 @@ sample:
 tests:
   sample.bluetooth.iso_broadcast_benchmark:
     build_only: true
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf5340_audio_dk_nrf5340_cpuapp
```

Comment:
We should also add CI builds for the `nrf5340_audio_dk_nrf5340_cpunet`

Reply:
Generally in samples where we already have `nrf5340dk_nrf5340_cpunet`, so something like `hci_rpmsg` and the `hci_uart` would make sense I think.

We may also want to copy the `.conf` files used for the nRF5340DK for those samples. @cvinayak what do you think?

Reply:
For CI builds, add the board under `integration_platforms` of the `hci_rpmsg` sample's `sample.yaml` for ISO Broadcast and ISO Receive configurations, here:

https://github.com/zephyrproject-rtos/zephyr/blob/ba810303c5cf489982b0c034d9db848217aaf7ea/samples/bluetooth/hci_rpmsg/sample.yaml#L12-L25

Reply:
Done